Project.toml

```diff
@@ -1,8 +1,14 @@
 name = "SimpleHypergraphs"
 uuid = "aa4a32ff-dd5d-5357-90e3-e7a9512f0501"
 authors = ["Bogumił Kamiński <bkamins@sgh.waw.pl>", "Przemysław Szufel <pszufe@sgh.waw.pl>"]
-version = "0.1.3"
+version = "0.1.4"
 
 [deps]
 LightGraphs = "093fc24a-ae57-5d10-9952-331d41423f4d"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
```

Comment:
Random and Test should go to `[extras]` section I think?

Reply:
This file is managed by the `Pkg` commands rather then edited manually. I tried your recommendation (moving to extras section) but it causes tests to fail.

Reply:
Do you run tests using `] test`?

Reply:
The `Project.toml ` specs is not fully documented. Any movement of the Random line causes the tetst chain to fails so it needs to stay as it is.